include/zephyr/bluetooth/audio/cap.h

```diff
@@ -636,6 +636,199 @@ struct bt_cap_broadcast_to_unicast_param {
 int bt_cap_initiator_broadcast_to_unicast(const struct bt_cap_broadcast_to_unicast_param *param,
 					  struct bt_bap_unicast_group **unicast_group);
 
+/**
+ * @brief Discovers audio support on a remote device.
+ *
+ * This will discover the Common Audio Service (CAS) on the remote device, to
+ * verify if the remote device supports the Common Audio Profile.
+ *
+ * @note @kconfig{CONFIG_BT_CAP_COMMANDER} must be enabled for this function. If
+ * @kconfig{CONFIG_BT_CAP_INITIATOR} is also enabled, it does not matter if
+ * bt_cap_commander_unicast_discover() or bt_cap_initiator_unicast_discover() is used.
+ *
+ * @param conn Connection to a remote server.
+ *
+ * @return 0 on success or negative error value on failure.
+ */
+int bt_cap_commander_unicast_discover(struct bt_conn *conn);
```

Comment:
This will effectively be identical to `bt_cap_initiator_unicast_discover`, unless we expand the CAP discovery function to discover the underlying services (e.g. VCS/MICS/BASS for commander and ASCS for Initiator)